tendrl/commons/flows/import_cluster/gluster_help.py

```diff
@@ -147,31 +147,40 @@ def import_gluster(parameters):
         )
         err, out, rc = command.run()
         if err:
+            err_msg = "Could not enable gluster-integration " \
+                "service. Error: %s" % err
             logger.log(
                 "error",
                 NS.publisher_id,
-                {
-                    "message": "Could not enable gluster-integration"
-                    " service. Error: %s" % err
-                },
+                {"message": err_msg},
                 job_id=parameters['job_id'],
                 flow_id=parameters['flow_id'],
             )
-            return False
+            return False, err_msg
 
     cmd = cmd_utils.Command(_cmd)
     err, out, rc = cmd.run()
     if err:
+        err_msg = "Could not start gluster-integration " \
+            "service. Error: %s" % err
         logger.log(
             "error",
             NS.publisher_id,
-            {
-                "message": "Could not start gluster-integration"
-                " service. Error: %s" % err
-            },
+            {"message": err_msg},
             job_id=parameters['job_id'],
             flow_id=parameters['flow_id'],
         )
-        return False
+        return False, err_msg
+
+    # Check if tendrl-gluster-integration active on the node
+    response = ServiceStatus(
+        "tendrl-gluster-integration"
+    ).status()
+    if not response:
+        return (
+            False,
+            "Service tendrl-gluster-integration not running on node: %s" %
```

Comment:
Please add to msg something like , cluster management requires service tendrl-gluster-integration

Reply:
Done